src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_fq_rnn_to_quantized_rnn.cpp

```diff
@@ -100,14 +96,28 @@ ov::intel_cpu::ConvertFqRnnToQuantizedRnn::ConvertFqRnnToQuantizedRnn() {
         const auto& r_weights    = pattern_map.at(R_m);
         const auto& bias         = pattern_map.at(B_m);
 
+        if (hidden_state.get_element_type() == element::i8) {
```

Comment:
Please leave a comment why we are doing this for i8

Reply:
Done